package.json

```
@@ -12,8 +12,19 @@
     "start-console": "./start-console.sh",
     "i18n": "./i18n-scripts/build-i18n.sh && node ./i18n-scripts/set-english-defaults.js",
     "ts-node": "ts-node -O '{\"module\":\"commonjs\"}'",
-    "lint": "eslint ./*.{js,ts} ./src && stylelint \"src/**/*.css\" --allow-empty-input",
-    "lint-fix": "eslint ./*.{js,ts} ./src --fix && stylelint \"src/**/*.css\" --allow-empty-input --fix"
+    "lint": "eslint ./*.{js,ts} ./src  ./tests && stylelint \"src/**/*.css\" --allow-empty-input",
+    "lint-fix": "eslint ./*.{js,ts} ./src ./tests --fix && stylelint \"src/**/*.css\" --allow-empty-input --fix",
```

We should also lint the `./cypress` dir

```suggestion
    "lint": "eslint ./*.{js,ts} ./cypress ./src ./tests && stylelint \"src/**/*.css\" --allow-empty-input",
    "lint-fix": "eslint ./*.{js,ts} ./cypress ./src ./tests --fix && stylelint \"src/**/*.css\" --allow-empty-input --fix",
```

---

jep, good idea. Triggered a bunch of stuff right away.